doc/md_src_files/055_decision-points_2.md

```diff
@@ -277,7 +277,7 @@ Because of this higher sensitivity to safety concerns, we chose to retain a four
 | Catastrophic    | Very High                 | Very High   | Very High |
 -->
 
-#### Adapting Situated Safety / Mission Impact for Sector-Specific Scenarios
+### Adapting Situated Safety / Mission Impact for Sector-Specific Scenarios
 
```

Comment:
How about:
Sector-Specific Mission Impact Scenarios

Reply:
The concept grouping here though is (situated safety and mission impact) for (sector-specific scenarios) so I don't think "Sector-Specific Mission Impact Scenarios" would work.

Reply:
### Situated Safety and Mission Impact Decision Points for Industry Sectors
?

Reply:
I like: 
Safety and Mission Impact Decision Points for Industry Sectors